src/index.html

```diff
@@ -5,8 +5,409 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#" class="is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a
+            href="#"
+            class="header-link"
+            data-qa="nav-hover">
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav-item">
+            <a href="#" class="header-link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="page">
+      <div class="catalog">
+        <div class="card" data-qa="card">
+          <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__product-image"
+          >
+
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <span class="card__product-code">
+            Product code: 195434
+          </span>
+
+          <div class="card__stars">
+            <div class="stars stars--4">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
```

Comment:
Checklist 2. [CODE STYLE] - Add empty lines between multiline sibling blocks of HTML. But don't add empty lines between parent and child elements